src/kinvey.js

```diff
@@ -184,6 +185,8 @@ Kinvey.AuthorizationGrant = AuthorizationGrant;
 Kinvey.CustomEndpoint = CustomEndpoint;
 Kinvey.DataStore = DataStore;
 Kinvey.DataStoreType = DataStoreType;
+Kinvey.Stream = Stream;
+Kinvey.LiveService = LiveServiceFacade;
```

Comment:
It's where user would register for live service (and get the pubnub auth keys and channel, etc.). It's also where they would attach the handlers for "global" pubnub events, such as disconnects.

I'm also thinking about attaching the `Stream` class to it, to make the connection more apparent. And if we are keeping the `StreamACL" class, it might get attached to it as well, or to the `Stream` class - haven't decided.